app/lib/data_transform/base.rb

```diff
@@ -0,0 +1,46 @@
+module DataTransform
```

Comment:
_non-blocking suggestion_: Since we're not using Rails in the most conventional way, I think it would help to include some brief documentation for any new modules, explaining the purpose and where they are meant to be used.

Reply:
Since it's non-blocking, I will add this in my next PR!